### DISCUSSION TITLE
Forwarding

### DISCUSSION THREAD

**Discussion by jbpin** [DNSCrypt/dnscrypt-proxy#2254] [Q&A]
Hi,

how can I configure forwarding rule with dynamically given dns address or forward to the system resolver ? 

I need to delegate some domain to the local dns but I don’t know the IP. This is the job of the DHCP to advertise the dns address. 

Thanks.

**Comment by jedisct1** [comment]
I don't think you can do that.

But even if the router acts as a DHCP server, its own address should always remain the same.

**Reply by jbpin** [reply]
YogaDNS allow to specify that the resolver would be the one given to the system. I understand that in dnscrypt-proxy you must change your dns to 127.0.0.1
